module/vision/LUTClassifier/src/BallEnhancer.cpp

```diff
@@ -247,23 +248,18 @@ namespace vision {
 
         // Loop through all of our possible ball segments
         std::vector<Eigen::Vector2i> points;
-        // NUClear::log("hSegments size = ", std::distance(hSegments.first,hSegments.second));
-        for (auto it = classifiedImage.horizontalSegments.begin(); it != classifiedImage.horizontalSegments.end();
-             ++it) {
-
+        for (const auto& segment : classifiedImage.horizontalSegments) {
             // We throw out points if they:
             // Have both edges above the green horizon
             // Are too small
-            if ((it->segmentClass == ClassifiedImage::SegmentClass::GOAL)
-                && (utility::vision::visualHorizonAtPoint(classifiedImage, it->start[0]) <= it->start[1]
-                    || utility::vision::visualHorizonAtPoint(classifiedImage, it->end[0]) <= it->end[1])
-                && it->length > 1) {
-                points.push_back(it->midpoint);
+            if ((segment.segmentClass == ClassifiedImage::SegmentClass::GOAL)
+                && (visualHorizonAtPoint(classifiedImage, segment.start[0]) <= segment.start[1]
+                    || visualHorizonAtPoint(classifiedImage, segment.end[0]) <= segment.end[1])
+                && segment.length > 1) {
+                points.push_back(segment.midpoint);
             }
         }
 
-        // std::vector<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>,
-        // Eigen::aligned_allocator<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>>> debug; // DEBUG LINE
```

Comment:
This is important! Keep it or make it better

Reply:
I'm pretty sure I just meant to move it onto the same line, so the `//DEBUG LINE` comments actually worked, but clang format got to it